RetailCoder.VBE/API/ParserState.cs

```diff
@@ -14,7 +14,56 @@
 namespace Rubberduck.API
 {
     [ComVisible(true)]
-    public interface IParserState
+    public enum ParserState
```

Comment:
I agree with ParserState.None being 0--except our state-evaluating code relies on it being greater than the rest of the values....  I'll rework that and make it equal 0.